Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -405,6 +426,10 @@ public void finish() {
         overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
     }
 
+    private void onVaultedPaymentMethodSelected(DropInEvent event) {
+        onVaultedPaymentMethodSelected(event.getPaymentMethodNonce(DropInEventProperty.VAULTED_PAYMENT_METHOD_SELECTION));
+    }
+
     public void onVaultedPaymentMethodSelected(final PaymentMethodNonce paymentMethodNonce) {
```

Comment:
Is this method called outside of DropInActivity? Can we make this private also and refactor the name? Not sure what name makes most sense here but it seems like we don't usually have overloaded method naming.

Reply:
Good call! It may have been necessary in the past before we had events. We can/should make this private.